code/modules/client/preferences.dm

```diff
@@ -529,6 +531,10 @@ GLOBAL_LIST_EMPTY(preferences_datums)
 			if(jobban_isbanned(user, rank))
 				HTML += "<font color=red>[rank]</font></td><td><a href='?_src_=prefs;jobbancheck=[rank]'> BANNED</a></td></tr>"
 				continue
+			var/available_in_playtime = job.available_in_playtime(user.client)
+			if(available_in_playtime)
+				HTML += "<font color=red>[rank]<//font></td><td><font color=red> \[ " + get_exp_format(available_in_playtime) + " as " + job.get_exp_req_type()  + "\] </font></td></tr>"

```

Comment:
why use + and not the [ ] syntax in byond strings? I don't remember exactly but I believe [ ] is faster than +, which is why https://github.com/tgstation/tgstation/pull/2283/files#diff-5cde94e4b6ce872079c39a1ac760f68cR65 was done.